### DISCUSSION TITLE
Support for pixart-ai under 8Gb of VRAM

### DISCUSSION THREAD

**Discussion by MOULINIER** [vladmandic/sdnext#2538] [Ideas]
Hi, could it be possible to implement the method used in the pixart GitHub to make the model able to work under 8Gb of VRAM ? 
I could try it but I'm scared to break things ( I'm not really experienced with python), would the push request be accepted?

**Comment by vladmandic** [comment]
PRs are more than welcome - I will review anything that is submitted and if it helps, it will be merged.
